pkg/apis/serving/register.go

```diff
@@ -33,6 +33,10 @@ const (
 	// pinned a revision
 	RevisionLastPinnedAnnotationKey = GroupName + "/lastPinned"
 
+	// RevisionPreservedAnnotationKey is the annotation key used for preventing garbage collector
+	// from automatically deleting the revision.
+	RevisionPreservedAnnotationKey = GroupName + "/preserved"
+
```

Comment:
I'd go more towards `exclude-gc` or `no-gc`.

Reply:
no-gc is fine.